guides/common/modules/con_managing-alternate-content-sources.adoc

```diff
@@ -2,9 +2,43 @@
 = Managing Alternate Content Sources
 
 Alternate content sources define alternate paths to download content during synchronization.
-The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+The content itself is downloaded from the alternate content source, while the metadata is downloaded from the {ProjectServer} or the upstream URL, depending on the configuration.
 You can use alternate content source to speed up synchronization if the content is located on the local filesystem or on a nearby network.
 You can set up alternate content sources for {ProjectServer} and {SmartProxy}.
 You must refresh the alternate content source after creation or after making any changes.
 A weekly cron job refreshes all alternate content sources.
 You can also refresh the alternate content sources manually using the {ProjectWebUI} or the Hammer CLI.
+
+There are three types of alternate content sources:
+
+Custom::
+Custom alternate content source downloads the content from any upstream repository or the local filesystem.
+
+Simplified::
+Simplified alternate content source copies the upstream repo information from your {ProjectServer}.
+You can use simplified alternate content sources in situations where the connection from your {SmartProxy} to the upstream repo is faster than from your {ProjectServer}.
+ifdef::satellite[]
+Selecting the Red Hat products when creating a simplified alternate content source will download the content to the {SmartProxies} from the {Team} CDN.
+endif::[]
+
+RHUI::
+RHUI alternate content source downloads content from the Red Hat Update Infrastructure server in your environment.
```

Comment:
To keep things consistent:

```suggestion
RHUI alternate content sources download content from a Red Hat Update Infrastructure server. Help with finding the correct network paths and importing authentication credentials is provided in the user interface.
```

Reply:
> Help with finding the correct network paths and importing authentication credentials is provided in the user interface.

@ianballou let me check with @maximiliankolb about adding this because as I see it, this part is very evident in the UI, so do we need to mention it in the docs explicitly? What do you think @maximiliankolb

Reply:
@maximiliankolb yep, here's [a link to the demo, the pertinent section starts at around 13:38](https://www.youtube.com/watch?v=k7naZLzgFro). Let me know what you think.

Reply:
Maybe change to

````
RHUI alternate content sources download content from a Red Hat Update Infrastructure server.
{ProjectWebUI} provides examples to help you find the network paths and to import authentication credentials.
````